js/src/renderer.ts

```diff
@@ -1,7 +1,8 @@
 import * as widgets  from '@jupyter-widgets/base';
 import {extend} from 'lodash';
 import * as pkg from '../package.json';
-import {Handsontable} from './handsontable';
+// @ts-ignore
```

Comment:
Ignoring with TypeScript (otherwise TypeScript complains that it does not exist)